event_model/__init__.py

```diff
@@ -396,10 +396,18 @@ def __init__(self, handler_registry, *,
         self._datum_cache = datum_cache
         self._descriptor_cache = descriptor_cache
         if retry_intervals is None:
-            retry_intervals = []
-        self.retry_intervals = list(retry_intervals)
+            self.retry_intervals = []
```

Comment:
It feels like it should be `self._retry_intervals = []`.
```suggestion
            self._retry_intervals = []
```